playbooks/common/openshift-cluster/disable_excluder.yml

```diff
@@ -3,9 +3,29 @@
   hosts: l_oo_all_hosts
   gather_facts: no
   tasks:
+
+  # During installation the excluders are installed with present state.
+  # So no pre-validation check here as the excluders are either to be installed (present = latest)
+  # or they are not going to be updated if already installed
+
+  # Install any excluder that is enabled
+  - include_role:
+      name: openshift_excluder
+      tasks_from: init
+  - include_role:
+      name: openshift_excluder
+      tasks_from: install
+    vars:
+      # Both docker_excluder_on and openshift_excluder_on are set in openshift_excluder->init task
+      install_docker_excluder: docker_excluder_on
+      install_openshift_excluder: openshift_excluder_on
```

Comment:
I don't think we should be referencing internal role variables outside of the role like this. As a side note, this is setting this variables to the strings `docker_exlcuder_on` and `openshift_excluder_on` rather than referencing the variables.